README.md

```diff
@@ -3,37 +3,130 @@
 [![Build Status](https://travis-ci.org/pingcap/pd.svg?branch=master)](https://travis-ci.org/pingcap/pd)
 [![Go Report Card](https://goreportcard.com/badge/github.com/pingcap/pd)](https://goreportcard.com/report/github.com/pingcap/pd)
 
-Pd is the abbreviate for placement driver.
+pd is the abbreviate for placement driver, used to manage and schedule [TiKV](https://github.com/pingcap/tikv) cluster. 
 
+Now pd itself support distributed and fault-tolerant through embedding [etcd](https://github.com/coreos/etcd). 
```

Comment:
PD supports distribution and fault-tolerance by embedding [etcd] (https://github.com/coreos/etcd).